templates/aspnet-s2i-template.json

```diff
@@ -280,7 +277,7 @@
         {
             "name": "ASP_UPSTREAM_IMAGE_TAG",
             "description": "Base OpenShift ASP S2I Image Tag",
-            "value": "latest",
+            "value": "1.0",
```

Comment:
Doesn't this mean that upgrades won't be automatically picked up? e.g. if you push `latest`, that'll be picked up, but not if you push the `1.0` tag again unless the ImagePullPolicy is Always